test/standalone/test-debuglet.js

```diff
@@ -44,25 +43,26 @@ var errorBp = {
   location: { path: 'fixtures/foo.js', line: 2 }
 };
 var fakeDebug = {
-  request: request
+  request: request // avoid authing.
```

Comment:
made the comment a bit more verbose. This is monkey patching the request method that the default Debug object has that goes through google-auth-library. This by-passes authentication for the purposes of this test.